assets/js/util/i18n.js

```diff
@@ -33,28 +33,51 @@ import { __, sprintf } from '@wordpress/i18n';
  * For example, passing 65 returns '1m 5s'.
  *
  * @since 1.0.0
+ * @since n.e.x.t Refactored and renamed to improve localization.
+ * @private
  *
- * @param {number} seconds The number of seconds.
- * @return {string} Human readable string indicating time elapsed.
- *
+ * @param {number}                     seconds   The number of seconds.
+ * @param {(Intl.NumberFormatOptions)} [options] Optional formatting options.
+ * @return {string}   		Human readable string indicating time elapsed.
  */
-export const prepareSecondsForDisplay = ( seconds ) => {
+const durationFormat = ( seconds, options = { unitDisplay: 'short' } ) => {
```

Comment:
Generally we always apply defaults in the destructuring, not in the default value itself. That way the individual defaults aren't lost if `options` are passed.

```suggestion
const durationFormat = ( seconds, { unitDisplay = 'short' } = {} ) => {
```

In this case, since we want all of the options in the one object it makes more sense to handle the defaults in the function, then we don't need to worry about passing `options.style` as well.

```js
const durationFormat = ( seconds, options = {} ) => {
	const options = {
		unitDisplay: 'short',
		...options,
		style: 'unit',
	};

```